database/schema.cql

```diff
@@ -0,0 +1,57 @@
+CREATE KEYSPACE place;
+
+CREATE TABLE place.place_by_id (
+  place_id uuid,
```

Comment:
I don't know if this is a major concern but I think you want to stay consistent with how you name the variables of either stick with place_id or placeId so here and your readme so it's consistent.

Reply:
I see. that is a good suggestion. 
I was thinking that place_id is for the database usage (I think DB doesn't like camelCase.) on the other hand, endpoints should be camelcase.